src/main/java/uk/gov/laa/pfla/auth/service/dao/ReportTrackingTableDao.java

```diff
@@ -25,19 +25,20 @@ public ReportTrackingTableDao(JdbcTemplate writeJdbcTemplate){
 
     public void updateTrackingTable(ReportTrackingTableModel trackingModel) {
 
+
         JdbcTemplate localJdbcTemplate = this.writeJdbcTemplate;
 
         String sql = "INSERT INTO GPFD.REPORT_TRACKING (ID, REPORT_NAME, REPORT_URL, CREATION_TIME, MAPPING_ID, REPORT_GENERATED_BY) VALUES (GPFD_TRACKING_TABLE_SEQUENCE.NEXTVAL,?,?,?,?,?)";
         Timestamp timestamp = Timestamp.valueOf(trackingModel.getCreationTime());
 
         //Insert values into sql statement and update
+        log.info("Updating tracking table");
```

Comment:
### From a security point of view Is it required to mention table name in the info logs?

Reply:
The name of the table is REPORT_TRACKING table, so we havent logged the table name? If i log  'updating **report** tracking information' like you suggest, thats arguably giving more information isnt it?

How about just 'updating tracking information' ?

Reply:
Looks good